src/tools/tests/goldens/kotlin-schema-generation.cgtest

```diff
@@ -203,6 +203,8 @@ arcs.core.data.Schema(
     refinement = { _ -> true },
     query = null
 )
+[require]
+test.results[0].match(/InlineEntity\("([a-f0-9]+)"\)/)[1] == test.results[1].match(/\n *"([a-f0-9]+)",\n/m)[1];
```

Comment:
Should we provide a top level `results` variable, so that we don't have to prefix it with `test.` every time?

Reply:
yeah, we can do that.